src/xml/Printer.cpp

```diff
@@ -314,10 +315,16 @@ std::ostream &printMD(std::ostream &out, const XMLTag &tag, int level, std::map<
 
       const auto ns = subtag->getNamespace();
       if (ns.empty()) {
-        out << "* [" << heading << "](" << link << ") `" << subtag->getOccurrenceString(subtag->getOccurrence()) << "`\n";
+        out << fmt::format("[* {}]({}) `{}`",
+                           heading,
+                           link,
+                           std::string(subtag->getOccurrenceString(subtag->getOccurrence())));
```

Comment:
This should work without conversion.
```suggestion
                           subtag->getOccurrenceString(subtag->getOccurrence()));
```